install/kubernetes/templates/istio-mixer.yaml.tmpl

```diff
@@ -89,6 +89,28 @@ spec:
         - NONE #--controlPlaneAuthPolicy
         - --customConfigFile
         - /etc/istio/proxy/envoy_mixer.json
+        - --zipkinAddress
+        - zipkin.istio-system:9411
```

Comment:
here, too. maybe run `grep` on istio-system to detect when it should be templatized?